lib/defaultMetrics.js

```diff
@@ -25,7 +26,8 @@ const metrics = {
 	processRequests,
 	heapSizeAndUsed,
 	heapSpacesSizeAndUsed,
-	version
+	version,
```

Comment:
using trailling comma style avoids this kind of diff churn, but that's a matter of project style, not for this PR

Reply:
(I'll put up a PR for it)

Reply:
Pls mark resolved.